etl/pypasar/omop/note.py

```diff
@@ -122,11 +122,11 @@ def process(self):
                             note_date,
                             (note_date::text || ' 00:00:00')::timestamp AS note_datetime, -- the function used to take date and join to midnight
                             32879 AS note_type_concept_id,
-                            0 AS note_class_concept_id,  -- Put 0 temporary as source_to_concept not included
+                            0 AS note_class_concept_id,  -- Put 0 as source_to_concept not found
                             note_title,
                             note_text, -- all value text is NULL
-                            32678 AS encoding_concept_id, -- clarification states data stored in database is UTF-8
-                            4180186 AS language_concept_id,
+                            0 AS encoding_concept_id, -- Put 0 as source_to_concept not found
+                            0 AS language_concept_id, -- Put 0 as source_to_concept not found
```

Comment:
@chowmunchun As we discussed previously, this has nothing to do with source_to_concept_map table rather the `concept` table itself where the record does exist in the concept table. But the reason this error occurred along with other clinical tables is because the column invalid_reason was set to `''` rather than null previously. However this has been fixed now.

Therefore please revert this change.

Please see the relevant SQL DQD Check below 
![image](https://github.com/user-attachments/assets/461ef2f5-3ad1-427f-9781-3829e53874e1)